components/automate-ui/src/app/page-components/navbar/navbar.component.html

```diff
@@ -7,7 +7,9 @@
       <div class="navigation-menu">
         <div role="menuitem" class="nav-link"><a routerLink="/dashboards/event-feed" routerLinkActive="active" tabindex="0">Dashboards</a></div>
 
-        <div role="menuitem" class="nav-link"><a routerLink="/applications/service-groups" routerLinkActive="active" tabindex="0">Applications</a></div>
+        <app-authorized [anyOf]="['/applications/service-groups', 'get']">
```

Comment:
I know this is still a draft, but you will probably want to check with the applications team to see what paths should be included in the `app-authorized`. It is not necessarily a one-to-one with the URL below. Looking at the applications tab, it could easily be that they need to have all permissions (`allOf`) for GetServices, GetServiceGroups, , and GetServiceGroupsHealthCounts, for example. Or it might be an `anyOf` for those. But it is the applications team that needs to inform you on that.

Reply:
Thanks @msorens - I'll reach out to the applications team tomorrow to clarify

Reply:
@vjeffrey - would you mind having a look at this and seeing if it covers what you intended?  otherwise I may need to add additional routes.